src/Discord.Net.WebSocket/Entities/Channels/SocketCategoryChannel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Discord.Audio;
+using Discord.Rest;
+using Model = Discord.API.Channel;
+
+namespace Discord.WebSocket
+{
+    [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
+    public class SocketCategoryChannel : SocketGuildChannel, ICategoryChannel
+    {
+        public override IReadOnlyCollection<SocketGuildUser> Users
+            => Guild.Users.Where(x => x.VoiceChannel?.Id == Id).ToImmutableArray();
+
```

Comment:
A `Channels` property should be implemented here since the cache is available to pull from.

Reply:
has been addressed, only card hasn't been closed since I added the feature under the selected line.